whenchanged/test_whenchanged.py

```diff
@@ -0,0 +1,91 @@
+#! /usr/bin/env python
```

Comment:
Please move unit tests to a `tests` folder in the root directory, and rename as `test_parse_args.py`